apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.cs

```diff
@@ -68,25 +68,42 @@ static SessionPool()
         }
 
         // ReSharper disable once UnusedMember.Global
-        internal static void DumpPoolContents(StringBuilder s)
+        // returns true if the pool has evenly split its contents.
+        // For test purposes only.
+        internal static bool GetIsPoolEvenlySplit(StringBuilder poolContents)
```

Comment:
Same comment here about the name only implying part of the methods purpose. Comment and/or change signature.